packages/gestalt/src/Box.js

```diff
@@ -136,27 +136,10 @@ type Props = {
    * Default: 'transparent'
    */
   color?:
```

Comment:
I believe the boxTransforms file also needs to be updated

Reply:
Done

Reply:
You'll also need to remove all current instances of the old colors from our docs examples, particularly in Box docs. See slack for details